src/outputFile.ts

```diff
@@ -0,0 +1,69 @@
+import * as fs from 'fs';
+import * as util from 'util';
+import * as path from 'path';
+
+const mkdirAsync = util.promisify(fs.mkdir);

```

Comment:
I believe you could do an `import * as fs from 'fs/promises'` to use the [promise version](https://nodejs.org/api/fs.html#fs_fs_promises_api) of these APIs.

Reply:
Only available on node >v10. Currently this extension is using 8.10.25 types. Do you know what version VS Code supports for extensions?

Reply:
The extension currently requires VS Code 1.40+, and it looks like VS Code 1.40 included Electron 7, which included Node 12.4.0. So we should be good to go to use `fs/promises`: https://code.visualstudio.com/updates/v1_40#_electron-60-update.

Reply:
Hm, I tried moving to using `fs/promises`, and am getting errors that the promisfied functions can't be found. It seems like they're not available in whatever node environment the extension is running in. Have you gotten this to work from a VS Code extension?

Reply:
Hmm, yeah I just tried it and it doesn't work either :/ Oh well, this is a minor change we can always make later!